.github/workflows/release.main.kts

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env kotlin
+@file:Repository("https://repo1.maven.org/maven2/")
+@file:DependsOn("io.github.typesafegithub:github-workflows-kt:2.3.0")
+
+@file:Repository("https://bindings.krzeminski.it/")
+@file:DependsOn("actions:checkout:v4")
+@file:DependsOn("actions:cache:v4")
+@file:DependsOn("actions:setup-java:v4")
+@file:DependsOn("gradle:actions__setup-gradle:v4")
+@file:DependsOn("nexus-actions:create-nexus-staging-repo:v1")
+@file:DependsOn("nexus-actions:release-nexus-staging-repo:v1")
+@file:DependsOn("nexus-actions:drop-nexus-staging-repo:v1")
+
+import io.github.typesafegithub.workflows.actions.actions.Cache
+import io.github.typesafegithub.workflows.actions.actions.Checkout
+import io.github.typesafegithub.workflows.actions.actions.SetupJava
+import io.github.typesafegithub.workflows.actions.gradle.ActionsSetupGradle
+import io.github.typesafegithub.workflows.actions.nexusactions.CreateNexusStagingRepo
+import io.github.typesafegithub.workflows.actions.nexusactions.DropNexusStagingRepoV1
```

Comment:
This is a bundled action binding, and it will be gone in the next version of the lib (in 2 weeks) 😊 Could we switch to the binding provided by the Maven-compatible server? The action provides the typings so in fact the only required changes are:

```suggestion
import io.github.typesafegithub.workflows.actions.nexusactions.DropNexusStagingRepo
```

and using this class in the workflow.

Reply:
Sure, will do